SeaBattle/Sea.cs

```diff
@@ -0,0 +1,92 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Runtime.CompilerServices;
+using System.Security.Cryptography.X509Certificates;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace SeaBattle
+{
+    public class Sea
+    {
+        private List<Deck> Decks { get; set; }
+        private List<BaseShip> Ships { get; set; }
+        int SeaWidth;
+        int SeaLenght;
+
+
+
+        public void Create()
+        {
+            //will finish this IF parts tommorow. 
+            //if ( > 10 &&  > 10)
+            //{
+            //}
+
+            for (int i = 0; i < Ships.Count; i++)
+            {
+                for (int j = 0; j < Decks.Count; j++)
+                {
+
+                }
+
+            }
+
+            var items = new List<PointStruct>(4);
```

Comment:
Still do not understand, why you creating all these ships inside the Sea.Create() method. It can't be handled from the outside, so you make it as a constant amount of ships, that will be created every time you call Create(). And it doesn't even stored anywhere...

Reply:
Deleted method create(), and did method which checks and adds ship to the sea.